project/Dependencies.scala

```diff
@@ -12,26 +12,26 @@ import sbt._
 object Dependencies {
 
   object Versions {
-    val crossScala = Seq("2.12.11")
+    val crossScala = Seq("2.13.3")
```

Comment:
To enable smooth transition across the project I would suggest cross-compiling the libraries for 2.12 and 2.13.